packages/airnode-deployer/src/infrastructure/__snapshots__/index.test.ts.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`listAirnodes lists Airnodes from multiple cloud providers 1`] = `
+"┌───────────────┬─────────────────┬───────────────────┬───────┬─────────────────┬───────────────────────────┐
+│ Deployment ID │ Cloud provider  │ Airnode address   │ Stage │ Airnode version │ Last update               │
```

Comment:
Shouldn't this be just "Version"? I think you discussed that with Andre somewhere...

Reply:
@bbenligiray mentioned that he would prefer just a `Version` label but I opted for `Airnode version` to stress that this is the version of the Airnode deployed not the version of the deployment, or the API or something.
We might expand this in the future though, I think the ChainAPI has a concept of API versiosn which might make sense displaying here.

Reply:
I see. There is a lot of room to bikeshed here :D 
I'd personally just call it `Airnode`.

Reply:
I agree with "Airnode version"